docs/source/crud/piccolo_crud.rst

```diff
@@ -244,6 +244,66 @@ For example, to return results 11 to 20:
 
 -------------------------------------------------------------------------------
 
+Bulk delete
+-----------
+
+To specify which records you want to delete in bulk, pass a query parameter 
+like this ``__ids=1,2,3``, and you be able to delete all results whose ``id`` 
+is in the query params.
+
+.. hint:: You can also use this method with ``UUID`` primary keys and 
+    the usage is the same.
+
+A query which delete movies with ``id`` pass in query parameter:
+
+.. code-block::
+
+    DELETE https://demo1.piccolo-orm.com/api/tables/movie/?__ids=1,2,3
+
+You can delete rows in bulk with any filter params. A query which 
+delete movies with ``name`` pass in query parameter:
+
+.. code-block::
+
+    DELETE https://demo1.piccolo-orm.com/api/tables/movie/?name=Star
+
+Or you can combine multiple query params for additional security. 
+A query to delete records with name ``Star``, but with 
+specific ``id`` you can pass query like this: 
+
+.. code-block::
+
+    DELETE https://demo1.piccolo-orm.com/api/tables/movie/?name=Star&__ids=1,2
+
+.. warning:: To be able to provide a bulk delete action, we must set 
+ ``allow_bulk_delete`` to ``True``.
+
+-------------------------------------------------------------------------------
+
+Bulk update
+-----------
+
+To specify which records you want to update in bulk, pass a query parameter 
+like this ``rows_ids=1,2,3``, and you be able to update all results whose ``id`` 
+is in the query params.
+
+.. hint:: You can also use this method with ``UUID`` primary keys and 
+    the usage is the same.
+
+A query which update movies with ``id`` pass in query parameter:
+
+.. code-block::
+
+    PATCH https://demo1.piccolo-orm.com/api/tables/movie/?rows_ids=1,2,3
+
+If you pass a wrong or non-existent value to the query parameters ``rows_ids``,
+no record will be changed and api response will be empty list.
```

Comment:
For bulk update, we're using a parameter called `row_ids`, and for bulk delete we're using a parameter called `__ids`. I think it would be better if they're the same (i.e. both are `__ids`).

Reply:
Can we stick to `rows_ids` (for both bulk delete and update) as I'm having trouble testing the FastAPI patch bulk in [here](https://github.com/sinisaos/piccolo_api/blob/6e072d29da05da5eb0284d98cf8c0389198bda89/piccolo_api/fastapi/endpoints.py#L278) with the `__ids` query parameter and I getting error. With `rows_ids` the test passes without problems.

Reply:
To make `ids` work, you might need to modify this so it's DELETE or PATCH:

https://github.com/sinisaos/piccolo_api/blob/6e072d29da05da5eb0284d98cf8c0389198bda89/piccolo_api/fastapi/endpoints.py#L483-L499